Empty heroku repository breaks vapor update

**Context**
swift 4.0.3
vapor toolbox 3.0.1, 3.1.4
git 2.14.3 (Apple Git-98)
empty repository on heroku
<details><summary>Package.swift</summary><p>

```swift
// swift-tools-version:4.0
import PackageDescription

let package = Package(
  name: "Bot",
  products: [
    .executable(name: "LevBot", targets: ["LevBot"]),
    ],
  dependencies: [
    .package(url: "https://github.com/leo150/Pelican.git", .branch("develop")),
    .package(url: "https://github.com/vapor/fluent-provider.git", .upToNextMajor(from: "1.2.0")),
    .package(url: "https://github.com/vapor-community/postgresql-provider.git", .upToNextMajor(from: "2.1.0"))
  ],
  targets: [
    .target(name: "LevBot",
        dependencies: [
          "Pelican",
          "FluentProvider",
          "PostgreSQLProvider"
        ],
        path: "Sources")
  ]
)
```
</p></details> <br />

**Trying**
 ```shell
$ vapor update
```

**Got**
>Error: backgroundExecute(code: 128, error: "fatal: ambiguous argument \'heroku/master...master\': unknown revision or path not in the working tree.\nUse \'--\' to separate paths from revisions, like this:\n\'git <command> [<revision>...] -- [<file>...]\'\n", output: "")

**Possible solution**
 ```shell
$ git remote remove heroku
$ vapor update
Updating [Done]
<...>
```
However, adding remote back breaks `update` again.
 ```shell
$ git remote add heroku <correct heroku url>
$ vapor update
Error: backgroundExecute <...>
```

Weird, I have the same bug, but I'm not on heroku but on gitlab as a remote origin. When I do a 
`mv .git .git2` and try `vapor build` again, I get the following output:
```
Error: backgroundExecute(code: 1, error: "warning: \'fetch\' command is deprecated; use \'resolve\' instead\n/Users/axel/github/artwhisper-server2: error: manifest parse error(s):\n/Users/axel/github/artwhisper-server2/Package.swift:3:22: error: incorrect argument label in call (have \'name:products:targets:dependencies:\', expected \'name:pkgConfig:targets:dependencies:\')\nlet package = Package(\n                     ^\n", output: "")
```

The  package.swift file was copied from a new vapor project, where it works flawlessly. 
I already remove .build, xcodeproject etc. to no avail.

Really weird bug.

--- a/Sources/VaporToolbox/Update.swift
+++ b/Sources/VaporToolbox/Update.swift
@@ -19,8 +19,6 @@ public final class Update: Command {
     }
 
     public func run(arguments: [String]) throws {
-        try checkGitUpstream()
-
         let isVerbose = arguments.isVerbose
         let bar = console.loadingBar(title: "Updating", animated: !isVerbose)
         bar.start()
@@ -34,37 +32,5 @@ public final class Update: Command {
             let xcode = Xcode(console: console)
             try xcode.run(arguments: arguments)
         #endif
-    }
-
-    func checkGitUpstream() throws {
-        guard gitInfo.isGitProject() else { return }
-        let currentBranch = try gitInfo.currentBranch()
-        
-        if let upstream = try? gitInfo.upstreamBranch() {
-            try gitInfo.verify(
-                local: currentBranch,
-                remote: upstream.remote,
-                upstream: upstream.branch
-            )
-        } else {
-            let remotes = try gitInfo.remoteNames()
-            let remote: String
-            if remotes.isEmpty {
-                return
-            } else if remotes.count == 1 {
-                remote = remotes[0]
-            } else if remotes.contains("origin") {
-                remote = "origin"
-            } else {
-                remote = try console.giveChoice(
-                    title: "Which remote are you tracking for '\(currentBranch)'?",
-                    in: remotes
-                )
-            }
-            try gitInfo.verify(
-                local: currentBranch,
-                remote: remote
-            )
-        }
     }
 }

